Engine/DataFeeds/LiveTradingDataFeed.cs

```diff
@@ -311,19 +311,62 @@ protected Subscription CreateDataSubscription(SubscriptionRequest request)
                 {
                     // tick subscriptions can pass right through
                     var tickEnumerator = new EnqueueableEnumerator<BaseData>();
-                    _exchange.SetDataHandler(request.Configuration.Symbol, data =>
+
+                    switch (request.Configuration.TickType)
```

Comment:
Seems to me we might be able to join the 3 cases together under the same behavior, thoughts? + extra check in the `if` for the `UpdateSubscriptionRealTimePrice` call for `tick.TickType != TickType.OpenInterest`.
I think it would be great if we could add a unit test to cover the issue being fixed